impeller/display_list/aiks_dl_vertices_unittests.cc

```diff
@@ -355,6 +355,10 @@ TEST_P(AiksTest, DrawVerticesPremultipliesColors) {
 }
 
 TEST_P(AiksTest, DrawVerticesWithInvalidIndices) {
+  if (GetBackend() == PlaygroundBackend::kOpenGLES) {
```

Comment:
Instead can you change this test to just asset that DisplayListToTexture returns a non-null ptr? Then we get all of the rendering/validation checks but dont have to check the golden.

Reply:
done